### ISSUE TITLE
getting-started.md improvements

### ISSUE THREAD

**Issue by bellwood** [digitalocean/netbox#24]
First off, my apologies this isn't in a pull request

1) "ALLOWED_HOSTS"

I had to add 127.0.0.1 as nginx is calling it for the proxy

2) gunicorn Configuration

You may want to elude to the command path needing to be checked, my path was sans local:

```
command = '/usr/bin/gunicorn'
```

**Comment by jeremystretch** [comment]
> I had to add 127.0.0.1 as nginx is calling it for the proxy

This shouldn't be necessary. Do you have the following under `location /` in your nginx config?

```
proxy_set_header X-Forwarded-Host $server_name;
```

> You may want to elude to the command path needing to be checked, my path was sans local

Looks like the default install location for Ubuntu changed recently. I'll update the doc to remove `local` from the path. Thanks for the heads up.

**Comment by bellwood** [comment]
@jeremystretch it looks like adding localhost isn't required - i had changed that and then removed local at the same time

removing 127.0.0.1 now and restarting supervisor leaves me with a working instance so I jumped the gun on that one

So far so good... I really like the work you've done here =)

**Comment by bellwood** [comment]
Looks like this was already addressed in 1cd2086 so I'll close this.

Thanks =)
